packages/apex-node/src/tests/codeCoverage.ts

```diff
@@ -117,56 +115,83 @@ export class CodeCoverage {
       return { codeCoverageResults: [], totalLines: 0, coveredLines: 0 };
     }
 
-    const codeCoverageResuls = await this.queryAggregateCodeCov(apexClassIdSet);
+    const codeCoverageAggregates = await this.queryAggregateCodeCov(
+      apexClassIdSet
+    );
 
     let totalLinesCovered = 0;
     let totalLinesUncovered = 0;
-    const codeCoverageResults: CodeCoverageResult[] = codeCoverageResuls.records.map(
-      item => {
-        totalLinesCovered += item.NumLinesCovered;
-        totalLinesUncovered += item.NumLinesUncovered;
-        const totalLines = item.NumLinesCovered + item.NumLinesUncovered;
-        const percentage = calculatePercentage(
-          item.NumLinesCovered,
-          totalLines
-        );
 
-        return {
-          apexId: item.ApexClassOrTrigger.Id,
-          name: item.ApexClassOrTrigger.Name,
-          type: item.ApexClassOrTrigger.Id.startsWith('01p')
-            ? 'ApexClass'
-            : 'ApexTrigger',
-          numLinesCovered: item.NumLinesCovered,
-          numLinesUncovered: item.NumLinesUncovered,
-          percentage,
-          coveredLines: item.Coverage.coveredLines,
-          uncoveredLines: item.Coverage.uncoveredLines
-        };
-      }
-    );
+    const totalCodeCoverageResults: CodeCoverageResult[] = [];
+
+    codeCoverageAggregates.forEach(chunk => {
+      const codeCoverageResults: CodeCoverageResult[] = chunk.records.map(
```

Comment:
Lines 128-151 in the new version are the same as lines lines 124-148 in the existing version.  The new code is wrapped in a `forEach()`, resulting in the code being indented.  Viewing with whitespace comparison turned off helps to see what really was changed.